### ISSUE TITLE
no need to calculate average after F.cross_entropy in loss.py

### ISSUE THREAD

**Issue by Vicky7777** [bhanML/Co-teaching#9]
In line 29 of loss.py,  there is no need to do torch.sum(loss_1_update)/num_remember. Because the result of  F.cross_entropy with default parameters reduce = True and size_average = True is the average loss.
